index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
> I mean, the only way to implement smooth scrolling is by disabling the browsers scrolling and implementing your own. If something in the scrolling mechanism has a bug, then you'll end up with no scrolling. Not sure it's a risk I'd like to have.

What do you mean? This smooth scrolling is about attaching it to anchor links on the same page, instead of having the page jump around when clicking the nav links in the menu. Great UX, if you ask me.

> It's also code that needs to be downloaded and parsed. The site currently takes more then 2 seconds to load, which is pretty long; not sure I'd like to patch more stuff onto that.

Nah, 2 seconds to *parse and render*. If you reload the page a couple of times on a warm cache, you as a user won't probably notice anything :) And removing this tiny lib will certainly not do anything for performance – just a micro optimisation. It's just a static site after all – have people complained about "performance issues"?